src/issues/issueFeatureRegistrar.ts

```diff
@@ -50,6 +51,28 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 
 	dispose() { }
 
+	async editQuery(query: vscode.TreeItem) {
+		const config = vscode.workspace.getConfiguration(ISSUES_CONFIGURATION);
+		const inspect = config.inspect<{ label: string, query: string }[]>(QUERIES_CONFIGURATION);
+		let command: string;
+		if (inspect?.workspaceValue) {
+			command = 'workbench.action.openWorkspaceSettingsFile';
+		} else {
+			command = 'workbench.action.openSettingsJson';
+		}
+		await vscode.commands.executeCommand(command);
+		const editor = vscode.window.activeTextEditor;
+		if (editor) {
+			const text = editor.document.getText();

```

Comment:
This is really cool! The "Configure Queries" action for changing the pull request queries just opens the settings page with prefilled search text, this seems much better. I'll create an issue to match this behavior with that action